### PR TITLE
update npm dependancies 

### PR DESCRIPTION
**Pull request by dtaylor7**
update npm dependancies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,12 +45,14 @@
...
     "joi": "^13.3.0",
+    "lodash": "^4.17.5",
     "moment": "^2.22.1",
     "os": "^0.1.1",
     "paths": "^0.1.1",
     "redis": "^2.8.0",
     "request": "^2.85.0",
-    "request-promise-native": "^1.0.5"
+    "request-promise-native": "^1.0.5",
+    "ua-parser-js": "^0.7.19"
```
Comment (hosainnet) [inline package.json]: Not sure why we're adding loadsh and ua-parser-js if the code hasn't changed?

Comment (tchow8) [reply package.json]: I think its a dependency of other modules we have, defining it here will enforce the version I believe?

Comment (dtaylor7) [reply package.json]: me neither, all i did was yarn upgrade. we are using lodash so must have not been added in this repo and used from a different

Comment (dtaylor7) [reply package.json]: must have somthing to do with resolutions

Comment (dtaylor7) [reply package.json]: removed ua-parser-js

**Pull request merged**